test/test-integration/src/test/resources/apis/openApis/api.yaml

```diff
@@ -0,0 +1,30 @@
+type: api
+version: v4
+data:
+  name: SwaggerPetstore
+  description: 'This is a sample server Petstore server.  You can find out more about
+    Swagger at [http://swagger.io](http://swagger.io) or on [irc.freenode.net, #swagger](http://swagger.io/irc/).  For
+    this sample, you can use the api key `special-key` to test the authorization filters.'
+  context: /v2/1.0.5
+  version: 1.0.5
+  provider: admin
+  lifeCycleStatus: CREATED
+  enableStore: true
+  type: HTTP
+  transport:
+  - http
+  - https
+  tags:
+  - pet
+  - store
+  - user
+  policies:
+  - Unlimited
+  visibility: PUBLIC
+  endpointConfig:
+    endpoint_type: http
+    production_endpoints:
+      url: http://localhost:8080
+    sandbox_endpoints:
+      url: http://localhost:8081
+  endpointImplementationType: ENDPOINT
```

Comment:
new line

Reply:
This file is temporary. The tests will be updated to use the downloaded apictl for project creation. Therefore, I think only the swagger files will remain.